airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/commands.py

```diff
@@ -53,6 +65,7 @@ async def build(ctx: click.Context, use_host_gradle_dist_tar: bool) -> bool:
         ctx.obj["concurrency"],
         ctx.obj["dagger_logs_path"],
         ctx.obj["execute_timeout"],
+        build_platforms,
```

Comment:
What if we added this to the context instead? Lets test and publish reuse it

Reply:
I don't think architecture should be customized for test and publish.
If we'd want to run multiarch tests I think it should be handled at the test level  and not a customizable input.
Same idea for publish, I think we should always build and publish for all our supported arch.

Reply:
Fair opinion on this not being something we want customizable for publish/test

But I was thinking of moving this to the context so that the entry point for the information of "which platforms are we building for?"

So that if you learn in `build` that the archicture target comes from the `context` you can take that knowledge with you to `publish` and `test`

Its not too far away from how other configurations like docker hub crednetials, cdk_version, connector_acceptance_test_image and others are in the context.

Reply:
@bnchrch I'm not sure which `context` you mean exactly.
Are we talking about the `ConnectorContext`, the `ctx.obj`, or the `ClickPipelineContext`?
I assume it's the `ConnectorContext` and I'll add it there. I can change it in a follow up PR if I'm wrong about your suggestion 😄